distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -220,7 +220,7 @@ public void testCswRegistryStoreUpdate() throws Exception {
                         FederationAdminService.class);
                 federationAdminServiceImpl.updateRegistryEntry(Library.getRegistryNode(regID,
                         "New Node Name",
-                        "2015-02-26T17:16:34.996Z"), destinations);
+                        "2016-02-26T17:16:34.996Z"), destinations);
```

Comment:
If 2015 failed, why does 2016 pass? Does it need to be within one year from test run? Is it something else fragile that will cause this test to fail next year? Should this date string be generated dynamically so it is future-proof?